CHARTER.md

```diff
@@ -0,0 +1,104 @@
+
+The Board shall have at least the following roles:
+
+ * Treasurer: Takes care of financial transactions and accounting.
+ * Secretary: Writes and publishes meeting notes, manages Voting membership.
+
+The Governing Board meets at least 4 times per year, publishes quarterly financial reports, and shares the content of their meetings with the MapLibre Community to the extent the Board deems appropriate.
+
+### Voting Members
+Voting Members are people who elect the Governing Board and who vote on changes to the MapLibre Charter.
+
+A precondition to becoming a Voting Member is that the person contributed in a non-trivial way or that they donated funds to the MapLibre Organization.
+
+If Voting Members are paid by a Third-party organization for their work on the MapLibre Projects, they shall declare publicly who their employer is. At most 10 percent of the Voting Members shall be employed by a single Third-party organization.
+
+Voting Members can nominate other Community members to receive Voting membership in acknowledgement of their contributions to the MapLibre Projects. The Voting Members then vote on granting the nominee a Voting membership. 
+
+Nominations for new Voting Members are open up to two weeks before the election of the Governing Board and the vote on granting Voting Membership to somebody takes place one week before the election of the Governing Board.
+
+### Governing Board Election
+Nomination process:
+
+ * A person can nominate themselves as a candidate for the Governing Board.
+ * Candidates have to register themselves up to 2 weeks before the election.
+
+Voting process:
+
+ * Governing Board elections take place every year.
+ * Each Voting Member can vote for up to 5 nominees with ranked voting principle.
+ * The nominee with the largest number of votes gets appointed, the nominee with the second largest number of votes gets appointed, and so on.
+ * In case there are multiple nominees per Third-party organization, only the one with the largest number of votes gets appointed.
+ * In case of a tie where more nominees than remaining seats receive an equal number of votes, the vote shall be repeated for the remaining seats.
+
+### Terms of Governing Board Members
+Governing Board members will serve for a one-year term.
+
+### Governing Board Member Remuneration
+Governing Board members receive no remuneration by the MapLibre Organization for their services on the Board, other than reimbursement for the out-of-pocket expenses incurred when carrying out activities on behalf of the Governing Board. Such expenses must be pre-authorized by at least two other board members.
+
+### Replacement of Governing Board Members
+In the event of a Governing Board member resigning from the Board during a term, the nominees from the last election with the largest number of votes shall be contacted with a proposal to serve in the Government Board  until the next election is called.
```

Comment:
nit: double space between "Government Board" and "until".

Reply:
@JannikGM just fyi, there is a suggestion box that would make this type of edit much easier -- 
![image](https://user-images.githubusercontent.com/1641515/183306059-5aba6b95-2adc-4a9c-89b3-13eef93e1875.png)

```suggestion
In the event of a Governing Board member resigning from the Board during a term, the nominees from the last election with the largest number of votes shall be contacted with a proposal to serve in the Government Board until the next election is called.
```